src/convertToObject.js

```diff
@@ -6,7 +6,23 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylesObject = {};
+  const stylesArray = sourceString.split(';');
```

Comment:
[CODE KNOWLEDGE] - There is no need to make a copy before each operation. Methods like `split` do not mutate the original string, so you can chain other methods directly after `split` without assigning it to an intermediate variable like 'stylesArray'.